web/gds-user-ui/src/components/NeedsAttention/index.tsx

```diff
@@ -38,9 +38,9 @@ const NeedsAttention = ({ buttonText, onClick }: NeedsAttentionProps) => {
         {attentionResponse?.messages?.map((item: AttentionResponseType, key: any) => (
           <AttentionAlert
             key={key}
-            action={item.action}
-            severity={item.severity}
-            message={item.message}
+            action={item?.action}
+            severity={item?.severity}
```

Comment:
The `severity` value is passed to the `status` prop of the `Alert` component. [When a status is passed as a prop](https://v1.chakra-ui.com/docs/components/feedback/alert#status) a preset color scheme and icon is applied to the `Alert` component. 

This is why the following error was returned: `Cannot read properties of undefined (reading 'colorScheme')`.

Adding optional chaining will check if item is undefined before attempting to access and pass the severity value.